lib/OrdinaryDiffEqDifferentiation/Project.toml

```
@@ -25,6 +29,7 @@ ADTypes = "1.11"
 ArrayInterface = "7"
 DiffEqBase = "6"
 DiffEqDevTools = "2.44.4"
+DifferentiationInterface = "0.6.23"
```

```suggestion
DifferentiationInterface = "0.6.28"
```
the other deps are also missing compat bounds?

---

```suggestion
DifferentiationInterface = "0.6.31"
```